### ISSUE TITLE
:isset modifier with @request.headers

### ISSUE THREAD

**Issue by ivanpajon** [pocketbase/pocketbase#5520]
I'm trying to create an API Rule that check some header are present in request.
I understand that :isset modifier is for @request.* so it should work with @request.headers.x_someheader (as per [documentation](https://pocketbase.io/docs/api-rules-and-filters/#isset-modifier))

But I was unable to set it because I get an error in UI:
![image](https://github.com/user-attachments/assets/51bf0d2c-f221-440a-bd5e-3f6f140b4d77)

I don't know if it's my mistake and I'm doing something wrong.

**Comment by ganigeorgiev** [comment]
Modifiers (aka. `:*` suffixed expressions) are not enabled for the `@request.headers.*` field at the moment. I'll consider either documenting it or enabling it with the next release.

As a workaround for now if you want to check whether a header has nonempty value you can compare it against empty string.

**Comment by ganigeorgiev** [comment]
I'm not sure how useful it would be but for consistency with the other `@request.*` fields I've enabled the modifiers for the headers field as part of the ongoing refactoring. The change will be available with the next v0.23.0 release soon (_there are no ETAs yet_).

**Comment by ivanpajon** [comment]
In my case I need to apply a filter in "users" collection like:

```
@request.headers.someheader:isset = true &&
@collection.somecollection.relationfield.name ?= @request.headers.someheader
```

So I filter users that have entries in "somecollection" with relationfield.name equals to someheader.
As the workaround you mentioned I'm trying with:

```
@request.headers.someheader != '' &&
@collection.somecollection.relationfield.name ?= @request.headers.someheader
```

But if I don't send the header it doesn't return any value.
